Makefile

```diff
@@ -106,7 +106,7 @@ ifdef CI
 	export GOCOVERDIR=test/coverage && \
 	if [ -d $${GOCOVERDIR} ]; then rm -r $${GOCOVERDIR}; fi && \
 	mkdir $${GOCOVERDIR} && \
-	gotestsum --junitfile integration-test-report.xml -- -v -race $$(go list ./... | grep github.com/confluentinc/cli/v3/test) && \
+	gotestsum --junitfile integration-test-report.xml -- -timeout 0 -v -race $$(go list ./... | grep github.com/confluentinc/cli/v3/test) && \
```

Comment:
Alternatively, we could just raise this to something like 15 or 20 minutes.

Reply:
Thought I did this a while ago... thanks!

Reply:
Can we add it to the other calls to `go test` just in case?

Reply:
Will do!